### ISSUE TITLE
Possible bug in geoprojection (Xmp.drone-dji.FlightYawDegree should be Xmp.drone-dji.GimbalYawDegree)

### ISSUE THREAD

**Issue by manuelrames** [DroneDB/DroneDB#321]
I have been trying the geoprojecting functionality (great work by the way) and I found that when doing so, the `Xmp.drone-dji.GimbalPitchDegree` and `Xmp.drone-dji.GimbalRollDegree` (so gimbal pitch and roll) are taken, which are correct, but in the case of the yaw angle, the tag `Xmp.drone-dji.FlightYawDegree` (corresponding to the UAVs yaw angle) is read. I tried geoprojecting using `Xmp.drone-dji.FlightYawDegree` and `Xmp.drone-dji.GimbalYawDegree` (by making `Xmp.drone-dji.FlightYawDegree = Xmp.drone-dji.GimbalYawDegree` value) and of course using the second approach yielded the correct geoprojection.

So, to sum up, my point is that https://github.com/DroneDB/DroneDB/blob/26337da740943a26e221ba052c520694bcc7ea48/src/exif.cpp#L402
might instead use the tag `Xmp.drone-dji.GimbalYawDegree`

**Comment by pierotofy** [comment]
Hey @manuelrames ✋ thanks for the report. Can you share a few test images?

**Comment by manuelrames** [comment]
Original image (no metadata modified, so `Xmp.drone-dji.FlightYawDegree` value is intact):

![DJI_20210824120352_0006_T](https://user-images.githubusercontent.com/14979496/182838478-95b9d8ce-2091-4078-9b63-3fe2a527922e.JPG)

which will yield an erroneus geoprojection. And another version of the same image in which `Xmp.drone-dji.FlightYawDegree` value has been modified and made equal to `Xmp.drone-dji.GimbalYawDegree`:

![DJI_20210824120352_0006_T_modified](https://user-images.githubusercontent.com/14979496/182841190-bb26c4c7-ee18-4684-8dd8-f26e1012207c.JPG)

Hopefully, geoprojecting both images should serve as a good example.

**Comment by pierotofy** [comment]
Thanks, it should be fixed with https://github.com/DroneDB/DroneDB/commit/9cf4a2b16ef1e1b7c185f0851b4dda7c7238af0a and will be available at the next release.

:pray:
